ariadne/src/Ariadne/Wallet/Cardano/Kernel/DB/HdWallet.hs

```diff
@@ -323,6 +341,8 @@ instance IxSet.Indexable (HdAddressId ': HdAddressIxs)
 -- We use a flat "relational" structure rather than nested maps so that we can
 -- go from address to wallet just as easily as the other way around.
 data HdWallets = HdWallets {
+  -- it should be IxSet (ixs :: [*]) (a :: *),
+  -- but it seems to be IxSet (a :: *)
```

Comment:
Because it's `IxSet` from `Ariadne.Wallet.Cardano.Kernel.DB.Util.IxSet`, not from the `ixset-typed` library.